pkg/sdk/file_format_test.go

```diff
@@ -146,17 +126,13 @@ func TestFileFormatsCreate(t *testing.T) {
 
 			FileFormatTypeOptions: FileFormatTypeOptions{
 				ParquetCompression:              &ParquetCompressionLzo,
-				ParquetSnappyCompression:        Bool(true),
```

Comment:
why was this removed ?

Reply:
Because this is not valid configuration (cant be used with ParquetCompression)